rllib/algorithms/algorithm_config.py

```diff
@@ -1965,26 +1965,26 @@ def learners(
 
         Args:
             num_learners: Number of Learner workers used for updating the RLModule.
-                A value of 0 means training will take place on a local Learner on main
+                A value of 0 means training takes place on a local Learner on main
                 process CPUs or 1 GPU (determined by `num_gpus_per_learner`).
                 For multi-gpu training, you have to set `num_learners` to > 1 and set
-                `num_gpus_per_learner` accordingly (e.g. 4 GPUs total and model fits on
+                `num_gpus_per_learner` accordingly (e.g., 4 GPUs total and model fits on
                 1 GPU: `num_learners=4; num_gpus_per_learner=1` OR 4 GPUs total and
                 model requires 2 GPUs: `num_learners=2; num_gpus_per_learner=2`).
             num_cpus_per_learner: Number of CPUs allocated per Learner worker.
                 Only necessary for custom processing pipeline inside each Learner
                 requiring multiple CPU cores. Ignored if `num_learners=0`.
             num_gpus_per_learner: Number of GPUs allocated per Learner worker. If
-                `num_learners=0`, any value greater than 0 will run the
-                training on a single GPU on the main process, while a value of 0 will
-                run the training on main process CPUs. If `num_gpus_per_learner` is
-                > 0, then `num_cpus_per_learner` should not be changed (from its default
+                `num_learners=0`, any value greater than 0 runs the
+                training on a single GPU on the main process, while a value of 0 runs
+                the training on main process CPUs. If `num_gpus_per_learner` is > 0,
+                then you shouldn't change `num_cpus_per_learner` (from its default
                 value of 1).
             local_gpu_idx: If `num_gpus_per_learner` > 0, and
-                `num_learners` < 2, then this GPU index will be used for
-                training. This is an index into the available
+                `num_learners` < 2, then RLlib uses this GPU index for training. This is
+                an index into the available
                 CUDA devices. For example if `os.environ["CUDA_VISIBLE_DEVICES"] = "1"`
-                then a `local_gpu_idx` of 0 will use the GPU with ID=1 on the node.
+                and `local_gpu_idx=0`, RLlib uses the GPU with ID=1 on the node.
```

Comment:
This feels counterintuitive. The GPU index 0 does not equal the environment variable 1 and we have two or more GPUs for a single learner. A user would expect a single GPU for a single learner when multiple GPUs are available on a node to be indicated with an ID or index. Do  I misunderstand sth here?